packages/data-provider/src/utils/formatUtils.ts

```diff
@@ -20,3 +20,7 @@ export function notEmpty<TValue>(value: TValue | null | undefined): value is TVa
 export function isNumber(value: any): value is number {
 	return typeof value === 'number';
 }
+
+export function isValidIdNumber(value: any) {
+	return isNumber(value) && !isNaN(value) && value > 0;
```

Comment:
A valid ID must be numeric greater than 0

Reply:
I like this. Your comment couldve been TSDoc ;)

Reply:
TSDocs added for clarification